shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -512,6 +515,11 @@ private boolean isAttached() {
         return mNativePlatformView != 0;
     }
 
+    void assertAttached() {
+        if (mNativePlatformView == 0)
```

Comment:
maybe if (!isAttached()) instead of if (mNativePlatformView == 0) to be consistent with the checks in other places.